recipe/build_pytorch.sh

```diff
@@ -89,24 +89,21 @@ export MAX_JOBS=${CPU_COUNT}
 
 if [[ ${cuda_compiler_version} != "None" ]]; then
     export USE_CUDA=1
-    export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX"
-    if [[ ${cuda_compiler_version} == 9.0* ]]; then
-        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;7.0"
-    elif [[ ${cuda_compiler_version} == 9.2* ]]; then
-        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0"
-    elif [[ ${cuda_compiler_version} == 10.* ]]; then
-        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
+    export TORCH_CUDA_ARCH_LIST="6.0+PTX;6.1;7.0"
```

Comment:
Can you keep this in sync with https://github.com/pytorch/builder/blob/30414521a665e04939a75e42da2aa4908be2fcd6/conda/pytorch-nightly/build.sh#L39 ?

Reply:
Done